File: fizzbuzz.cpp
/*****************************
*         FIZZBUZZ-C++       *
*        By Sage Smith       *
* sagemistressmods@gmail.com *
******************************/
// kopimi

#include <iostream>
#include <array>

using namespace std;

int main()
{
    int x = 1;
    array<int,100> numbers;

    for (int i = 0; i < numbers.size(); i++)
    {
        numbers[i] = x;
        x++;
    }

    for (int i = 0; i < numbers.size(); i++)
    {
        if (numbers[i] % 3 == 0)
        {
            if (numbers[i] % 5 == 0)
            {
                cout << "FizzBuzz\n";
                continue;
            }
            cout << "Fizz\n";
            continue;
        }
        else if (numbers[i] % 5 == 0)
        {
            cout << "Buzz\n";
            continue;
        } 
        else
        {
            cout << numbers[i] << endl;
        }
    }
    
    return 0;
}

File: fizzbuzz.java
/*****************************
*       FIZZBUZZ-JAVA        *
*       By Sage Smith        *
* sagemistressmods@gmail.com *
******************************/
// kopimi

class Fizzbuzz
{
    public static void main(String[] args)
    {
        int x = 1;
        int[] numbers = new int[100];

        for (int i = 0; i < numbers.length; i++)
        {
            numbers[i] = x;
            x++;
        }

        for (int i = 0; i < numbers.length; i++)
        {
            if (numbers[i] % 3 == 0)
            {
                if (numbers[i] % 5 == 0)
                {
                    System.out.println("FizzBuzz");
                    continue;
                }
                System.out.println("Fizz");
                continue;
            }
            else if (numbers[i] % 5 == 0)
            {
                System.out.println("Buzz");
                continue;
            } 
            else
            {
                System.out.println(numbers[i]);
            }
        }
    }
}
